app/api/routers/breeze_buddy/telephony/inbound/handlers.py

```
@@ -185,68 +148,369 @@ async def handle_voicebot_url(request: Request) -> Response:
+    logger.info(f"[Plivo:XML] WebSocket URL (raw): {ws_url}")
+    logger.info(f"[Plivo:XML] WebSocket URL (escaped): {ws_url_escaped}")
+    logger.info(f"[Plivo:XML] Noise cancellation enabled: {noise_cancellation_enabled}")
+    logger.info(f"[Plivo:XML] Noise cancellation level: {noise_cancellation_level}")
+    logger.info(f"[Plivo:XML] Generated Stream XML:\n{xml}")
+
+    return xml
+
+
+def build_plivo_ivr_xml(
+    template_list: list,
+    voice_name: str,
```

Ruff (0.15.0)

[warning] Unused function argument: `voice_name`

(ARG001)